From: Karl Berry <karl@freefriends.org>
Subject: Remove (incompatible) declarations from c-memstr.h.

 * c-memstr.h (strtok, strstr): remove () fallback declarations,
   since C23 considers that the same as (void). Apparently
   HAVE_STRING_H was not defined, either.
   Report from Hilmar, 3 Apr 2025 23:54:54.

Origin: upstream, https://svn.tug.org:8369/texlive?revision=74888&view=revision
Bug-Debian: http://bugs.debian.org/1109256

Index: texk/kpathsea/c-memstr.h
===================================================================
--- texlive-bin.orig/texk/kpathsea/c-memstr.h	(revision 74887)
+++ texlive-bin/texk/kpathsea/c-memstr.h	(revision 74888)
@@ -53,11 +53,4 @@
 #define memcpy(to, from, len) bcopy ((from), (to), (len))
 #endif
 
-#if !defined(HAVE_STRING_H)
-extern char *strtok ();
-#ifndef strstr
-extern char *strstr ();
-#endif
-#endif
-
 #endif /* not KPATHSEA_C_MEMSTR_H */
